test/crud-with-options.test.js

```diff
@@ -272,6 +272,28 @@ describe('crud-with-options', function() {
       User.find({limit: 3});
     });
 
+    it('should allow filter with groupBy, count, max, min, sum & avg', function(done) {
```

Comment:
can you add test cases for when there are other filter queries too and limit and offset is also provided ? Basically to confirm whether this feature works with other features together or not. And whether it has any impact there.

Reply:
Hi, @samarpanB. I have added one more test to verify the change with existing filters.